javascripts/htmlBuilder.js

```diff
@@ -0,0 +1,51 @@
+
+//**********//HTML BUILDER//**********//
+var petFood = function(object) {
+	
+	//DECLARES VARIABLE FOR OUTPUT DOM ELEMENT//
+	var mainContent = document.getElementById("mainContent");
+
+	//BUILDS HTML CONTENT FROM ARRAY PARAMETER FOR A GIVEN ANIMAL//
+	object.htmlBuilder = function(array, foodAnimal) {
+
+		//ADDS FOOD ANIMAL HEADER//
+		content = `<ul class="foodAnimal">`
+		content += `<li><h1>${foodAnimal.toUpperCase()}</h1></li>`;
+
+		//ADDS BRAND NAME//
+		for (var i = 0; i < array.length; i++) {
+			content += '<ul class="foodBrand">';
+			content += `<li><h2>${array[i].name}</h2></li>`;
+
+			//ADDS LIST OF BREEDS FOR CAT FOOD//
+			if(foodAnimal === "cat") {
+				content += `<li><b>Breeds:</b> ${array[i].breeds.join(", ")}</li>`;
+			};
+
+			//ADDS FOOD TYPE TITLES//
+			for (var c = 0; c < array[i].types.length; c++) {
+				content += '<ul class="foodType">';
+				content += `<li><h3>${array[i].types[c].type}</h3></li>`;
```

Comment:
...this can just be ```type.type``` etc.  Definitely helps in readability and saves from the possibility of introducing errors when mistyping a huge long chained set of looped arrays.